palace.pyx

```diff
@@ -1208,6 +1212,117 @@ cdef class SourceGroup:
         """
         self.impl.destroy()
 
+
+cdef class AuxiliaryEffectSlot:
+    """An effect processor.
+
+    It takes the output mix of zero or more sources,
+    applies DSP for the desired effect (as configured
+    by a given `Effect` object), then adds to the output mix.
+
+    Parameters
+    ----------
+    context : Context
+        The context from which the auxiliary effect slot is to be created.
+
+    Raises
+    ----------
+    RuntimeError
+        If the effect slot can't be created.
+    """
+    cdef alure.AuxiliaryEffectSlot impl
+
+    def __init__(self, context: Context) -> None:
+        self.impl = (<Context> context).impl.create_auxiliary_effect_slot()
+
+    def __enter__(self) -> AuxiliaryEffectSlot:
+        return self
+
+    def __exit__(self, exc_type: Optional[Type[BaseException]],
+                 exc_val: Optional[BaseException],
+                 exc_tb: Optional[TracebackType]) -> Optional[bool]:
+        self.destroy()
+
+    def __lt__(self, other: Any) -> bool:
+        if not isinstance(other, AuxiliaryEffectSlot):
+            return NotImplemented
+        return self.impl < (<AuxiliaryEffectSlot> other).impl
+
+    def __le__(self, other: Any) -> bool:
+        if not isinstance(other, AuxiliaryEffectSlot):
+            return NotImplemented
+        return self.impl <= (<AuxiliaryEffectSlot> other).impl
+
+    def __eq__(self, other: Any) -> bool:
+        if not isinstance(other, AuxiliaryEffectSlot):
+            return NotImplemented
+        return self.impl == (<AuxiliaryEffectSlot> other).impl
+
+    def __ne__(self, other: Any) -> bool:
+        if not isinstance(other, AuxiliaryEffectSlot):
+            return NotImplemented
+        return self.impl != (<AuxiliaryEffectSlot> other).impl
+
+    def __gt__(self, other: Any) -> bool:
+        if not isinstance(other, AuxiliaryEffectSlot):
+            return NotImplemented
+        return self.impl > (<AuxiliaryEffectSlot> other).impl
+
+    def __ge__(self, other: Any) -> bool:
+        if not isinstance(other, AuxiliaryEffectSlot):
+            return NotImplemented
+        return self.impl >= (<AuxiliaryEffectSlot> other).impl
+
+    def __bool__(self) -> bool:
+        return <boolean> self.impl
+
+    def set_gain(self, gain: float) -> None:
+        return self.impl.set_gain(gain)
+
+    def set_send_auto(self, sendauto: bool) -> None:
+        return self.impl.set_send_auto(sendauto)
```

Comment:
For these two setters, please omit `return` and use `value` instead of property's name.  There's absolutely no reason for this, just to be consistent with the rest of the code.